### PR TITLE
Custom Title Bar for Windows

### PR DESCRIPTION
**Pull request by artsiom-voitas**
* Implemented custom title bar for Windows
* Updated configuration for Tauri v2 support
* Added couple of keys to i18n
* Minor changes to some locales

### CODE REVIEW

**Comment by artsiom-voitas** [comment]
Before image

After image

**Comment by utkubakir** [comment]
@ameer2468 can you test this on windows?

File: interface/locales/it/common.json [hunk 1]
```diff
@@ -393,6 +394,7 @@
   "meet_title": "Incontra {{title}}",
   "mesh": "Mesh",
   "miles": "Miglia",
+  "minimize": "Minimize",
```
Comment (utkubakir) [inline interface/locales/it/common.json:397]: can we remove non-translated keys?

Comment (artsiom-voitas) [reply interface/locales/it/common.json:397]: > can we remove non-translated keys?

Sure, will do right now,

**Comment by ameer2468** [comment]
> @ameer2468 can you test this on windows?

Tested and it works great!

**Review by utkubakir — APPROVED** [review]
thank you

**Comment by artsiom-voitas** [comment]
> thank you

Glad to make the Windows version more pleasing to the eye.
Press W for Windows 😃

**Pull request merged**